iXBRLViewerPlugin/viewer/src/js/inspector.js

```diff
@@ -136,7 +136,10 @@ Inspector.prototype.initializeViewer = function () {
     viewer.onMouseLeave.add(id => this.viewerMouseLeave(id));
     $('.ixbrl-next-tag').click(() => viewer.selectNextTag(this._currentItem));
     $('.ixbrl-prev-tag').click(() => viewer.selectPrevTag(this._currentItem));
-    this.search();
+}
+
+Inspector.prototype.postLoadAsync = function () {
+    runGenerator(this._search.buildSearchIndex(() => this.searchReady()));
```

Comment:
nit: this is fine, but could also use `this.searchReady` directly.
```suggestion
    runGenerator(this._search.buildSearchIndex(this.searchReady));
```

Reply:
I don't think this works, as `searchReady()` will be invoked with `this` undefined.  We need the closure to ensure that the method is invoked with the right object as `this`.